text/0630-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: https://github.com/emberjs/rfcs/pull/630
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
```

Comment:
```suggestion
# Document Browser Support Policy
```
Recommend changing this to document the reality of the current browser support (so it can be policy that can be reviewed/changed at an official/project level). This sets the project up for more flexibility and future growth.

Reply:
The current browser support policy is poorly documented, I agree. However, what I am proposing is a stark departure from current policy. I will document current policy in this RFC for comparison.

Reply:
Sure, but regardless this RFC is attempting to explicitly document the _new_ policy (while deciding what it is).

Reply:
@MelSumner @rwjblue Documented current policy. Please rereview.